cpp/include/cudf/detail/binaryop.hpp

```diff
@@ -79,4 +79,50 @@ std::unique_ptr<column> binary_operation(
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
 }  // namespace detail
+
+namespace experimental {
+namespace detail {
+/**
+ * @copydoc cudf::experimental::binary_operation(scalar const&, column_view const&, binary_operator,
+ * data_type, rmm::mr::device_memory_resource *)
+ *
+ * @param stream CUDA stream used for device memory operations and kernel launches.
```

Comment:
Parameter order here is incorrect.  Should be `stream, mr`

Same applies for the other 3 declarations in this file.

Reply:
`@copydoc` is used at many places in our cpp documentation to avoid duplication of doc text, most often used in detail APIs. There is no cleaner way so far. 

Note: Alternatively, we could contribute to doxygen to reorder the `@param` as listed in the function signature. Timeline: Uncertain.

Reply:
> 
> 
> `@copydoc` is used at many places in our cpp documentation to avoid duplication of doc text, most often used in detail APIs. There is no cleaner way so far.
> 

Right, but isn't this going to result in incorrect documentation?  My expectation is that you will end up getting something like:

```
 * @param lhs         The left operand scalar
 * @param rhs         The right operand column
 * @param op          The binary operator
 * @param output_type The desired data type of the output column
 * @param mr          Device memory resource used to allocate the returned column's device memory
 * @param stream    stream CUDA stream used for device memory operations and kernel launches.
 * @return            Output column of `output_type` type containing the result of
 *                    the binary operation
 * @throw cudf::logic_error if @p output_type dtype isn't fixed-width
```

...which is the incorrect order.

Reply:
This is how it's rendered now.
![image](https://user-images.githubusercontent.com/6488848/125840397-860c251d-17dc-46aa-833d-64f91dfb9e79.png)

Reply:
This is not better too.

Reply:
detail APIs are not in released libcudf documentation. They are excluded by doxygen.